recipes/pymol-open-source/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "2.4.0" %}
+{% set number = "0" %}
+{% set sha256 = "5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581" %}
+
+package:
+  name: pymol-open-source
+  version: {{ version }}
+
+source:
+  url: https://github.com/schrodinger/pymol-open-source/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ number }}
+  script: "{{ PYTHON }} setup.py install --use-msgpackc=no --no-libxml --no-vmd-plugins"
+  skip: True  # [py < 37]
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - python
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - mesalib                        # [linux]
+    - libglu                         # [linux]
+    - libpng
+    - freetype
+    - pyqt
+    - glm
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+  host:
+    - python
+    - pip
+    - glew
+    - libglu                         # [linux]
+    - libpng
+    - freetype
+    - glm
+    - xorg-libxfixes
+  run:
+    - python
+    - numpy
+    - libpng
+    - freetype
+    - pyqt
+    - glm
+
+test:
+  files:
+    - 1aki.cif
+    - test.py
+  commands:
+    - "python test.py"
+
+about:
+  home: https://pymol.org/
+  license: LicenseRef-PyMOL
+  license_file: LICENSE.txt
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
The problem are the linked libraries, whose license needs to be referenced too, as far as I know.